nats/subsz.yml

```diff
@@ -0,0 +1,60 @@
+type: json
+data:
+  url: /subsz
+  server: ${NATS_HOSTPORT}
+
+observation:
+  - name: subsz
+    metricNamespace: nats
+    path: $.
+
+    metric:
+      - name: subscriptions
+        source: num_subscriptions
+        type: long_gauge
+        label: subscriptions
+        description: The number of subscriptions
+
+      - name: inserts
```

Comment:
Maybe rename to `subscriptions.inserts`, to make it clear these are subscriptions related? Renaming label to `subscriptions list inserts` would maybe also be OK. And the same for other metrics below.

Reply:
OK. DONE